playbooks/roles/redis-exporter/README.md

```diff
@@ -0,0 +1,14 @@
+redis-exporter
+==============
+
+Extends prometheus_redis_exporter_role to be used in Ocim.
+
+Role Variables
+--------------
+
+A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
```

Comment:
Remove boilerplate text and add actual details about the role variables or add a note to check `defaults/main.yml`.